cpp/src/IceSSL/SSLEngine.cpp

```diff
@@ -70,7 +76,7 @@ IceSSL::SSLEngine::initialize()
 }
 
 void
-IceSSL::SSLEngine::verifyPeerCertName(const string& address, const ConnectionInfoPtr& info)
+IceSSL::SSLEngine::verifyPeerCertName(const ConnectionInfoPtr& info, const string& address) const
```

Comment:
This should probably be a `string_view`?

Reply:
I want to move this to the Schannel implementation, which is the only one using it. But let's do it in a follow-up PR.

Reply:
See #2111